src/analyzer/analyzer.csproj

```diff
@@ -7,15 +7,10 @@
     <LangVersion>latest</LangVersion>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(MonoBuild)' == ''">
-    <DefineConstants>$(DefineConstants);FEATURE_ILLINK</DefineConstants>
+  <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>
```

Comment:
nit: combine this with the previous PropertyGroup

Reply:
I tried not to change all csproj much, they all deserve another cleanup